netci.groovy

```diff
@@ -77,8 +77,7 @@ Constants.scenarios.each { scenario ->
 
                 // This call performs test run checks for the CI.
                 Utilities.addXUnitDotNETResults(newJob, '**/testResults.xml')
-                Utilities.addArchival(newJob, "${workspaceRelativeFxRootLinux}/bin/**/testResults.xml")
-                Utilities.addXUnitDotNETResults(newJob, "${workspaceRelativeFxRootLinux}/bin/**/testResults.xml")
+                Utilities.addArchival(newJob, "**/testResults.xml")
```

Comment:
Do we need to keep the `addArchival` that is under `scenario == corefx` 5 lines below this?